remote.c

```diff
@@ -2226,6 +2370,14 @@ execute_cmd(struct daemon_remote* rc, RES* ssl, char* cmd, struct rc_state* rs)
 		do_assoc_tsig(ssl, rc->xfrd, skipwhite(p+10));
 	} else if(cmdcmp(p, "del_tsig", 8)) {
 		do_del_tsig(ssl, rc->xfrd, skipwhite(p+8));
+	} else if(cmdcmp(p, "add_cookie_secret", 17)) {
+		do_add_cookie_secret(ssl, rc->xfrd, skipwhite(p+18));
+	} else if(cmdcmp(p, "drop_cookie_secret", 18)) {
+		do_drop_cookie_secret(ssl, rc->xfrd, skipwhite(p+18));
+	} else if(cmdcmp(p, "print_cookie_secrets", 20)) {
+		do_print_cookie_secrets(ssl, rc->xfrd, skipwhite(p+20));
+	} else if(cmdcmp(p, "activate_cookie_secret", 22)) {
+		do_activate_cookie_secret(ssl, rc->xfrd, skipwhite(p+18));
```

Comment:
```suggestion
		do_activate_cookie_secret(ssl, rc->xfrd, skipwhite(p+22));
```
The string length is 22.